dashboard/app/models/levels/blockly.rb

```diff
@@ -504,25 +504,57 @@ def localized_function_blocks(blocks)
         default: nil,
         smart: true
       )
-      function_description = function.at_xpath('./mutation/description')
+      original_function_name = function_name.content
+      function_name.content = localized_name if localized_name
+      function_mutation = function.at_xpath('./mutation')
```

Comment:
Could you leave an inline comment explaining what this "mutation" stuff is, if you know what it is. It might be helpful for future developers when they try to understand what this code does.